.github/blog-template-demo.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: ''
+Author: Your name here
+Author handle: Any links you want associated with your name
+Date: ''
+Description: one line about your blog
+Folder with media files: ''
```

Comment:
What's the use of this parameter? Shouldn't these just be extra files in the PR?